src/data/real/ennreal.lean

```diff
@@ -1337,6 +1337,30 @@ end
 @[simp] lemma one_sub_inv_two : (1:ℝ≥0∞) - 2⁻¹ = 2⁻¹ :=
 by simpa only [div_eq_mul_inv, one_mul] using sub_half one_ne_top
 
+/-- Order isomorphism between `ℝ≥0∞` and the unit interval `set.Iic (1 : ℝ≥0∞)`. -/
+def order_iso_Iic_1 : ℝ≥0∞ ≃o Iic (1 : ℝ≥0∞) :=
```

Comment:
`@[simps]` seems sensible here.

Is there a name you could give this that more accurately describes the specific map you're choosing?